src/index.html

```diff
@@ -22,6 +22,375 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="image"
+        src="././images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="navigation">
+        <ul class="list">
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link is-active"
```

Comment:
Fix class for all links
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```